lib/Interactions/Internal/WebDriverSingleKeyAction.php

```diff
@@ -20,5 +27,9 @@ public function __construct(
     ) {
         parent::__construct($keyboard, $mouse, $location_provider);
         $this->key = $key;
+
+        if (!in_array($key, self::MODIFIER_KEYS, true)) {
+            throw new \InvalidArgumentException("Key Down / Up events only make sense for modifier keys.");
```

Comment:
I would slightly reword the exception to something like: 

```suggestion
            throw new \InvalidArgumentException('keyDown / keyUp actions can only be used for modifier keys');
```